setup.py

```diff
@@ -60,7 +60,8 @@ def find_files(directory):
           'pyFai',
           'PyQt5 >= 5.13.2, <= 6.0.0',
           'PyQt5-sip >= 12.7.0, <= 13.0.0',
-          'scipy'
+          'scipy',
+          'xarray <= 0.14.1'
```

Comment:
```suggestion
          'xarray <= 0.14.1',
```

Trailing commas are nice, then the next addition only needs to change one line.